configs/tsconfig-base.json

```diff
@@ -13,7 +13,6 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "pretty": true,
-    "sourceMap": true,
```

Comment:
Ouch, forgot this. I removed this because sourcemap handling is different in gulp(-ts).